bottomsheet/src/main/java/com/flipboard/bottomsheet/BottomSheetLayout.java

```diff
@@ -450,11 +450,13 @@ private boolean canScrollUp(View view, float x, float y) {
             ViewGroup vg = (ViewGroup) view;
             for (int i = 0; i < vg.getChildCount(); i++) {
                 View child = vg.getChildAt(i);
-                int childLeft = child.getLeft();
-                int childTop = child.getTop();
-                int childRight = child.getRight();
-                int childBottom = child.getBottom();
-                boolean intersects = x > childLeft && x < childRight && y > childTop && y < childBottom;
+
+                int childLeft = child.getLeft() - view.getScrollX();
+                int childTop = child.getTop() - view.getScrollY();
+                int childRight = child.getRight() - view.getScrollX();
+                int childBottom = child.getBottom() - view.getScrollY();
+
+                boolean intersects = childLeft < x && x < childRight && childTop < y && y < childBottom;
```

Comment:
Please undo the changes here, and undo the added newlines. They're not functional.